src/Export_Grouped_Timestamps_To_CSV.py

```diff
@@ -0,0 +1,82 @@
+import os
+import h5py
+from datetime import datetime, timezone
+from dateutil import parser
+import csv
+
+
+class Export_Grouped_Timestamps_To_CSV:
+
+    def __init__(self, folder_path):
+        self.folder_path = folder_path
+        self.all_timestamps = []
+        self.timestamps_by_decade = {}
+
+    def read_files(self):
+        files = os.listdir(self.folder_path)
+        for file in files:
+            file_path = os.path.join(self.folder_path, file)
+            if file.endswith('.h5'):
+                self.extract_and_print_timestamps(file_path)
```

Comment:
Die Funkton gibt es doch schon, muss die hier noch einmal erstellt werden? Es reicht doch wenn du die Funktion aufrufst.

Reply:
Die gibt es nicht schon so. die wurde erweitert um:
 self.all_timestamps = []
        self.timestamps_by_decade = {}

Reply:
@Fabiancomida es gibt drei Methoden die read files heißen unterschied ist das darin eine andere Methode aufgerufen wird das ist eigentlich unnötig. Du initialisierst nur in der Klasse zwei weitere Vaiablen. Das ist aber auch nicht notwendig, wenn du die Variablen nicht übergibst.

Reply:
ich baue das um und nutze stattdessen den filereader...

Reply:
Die drei Punkte machen das ja ganz dramatisch kleiner Python-Dev